workbench/config/defaults.php

```diff
@@ -544,7 +545,7 @@
         "default" => true,
         "overrideable" => true,
         "dataType" => "boolean",
-        "minApiVersion" => 40.0
+        "minApiVersion" => 41.0
```

Comment:
Did you mean to change this line? 

@john-brock is 41 needed for streaming v2?